pgbouncer/values.yaml

```diff
@@ -62,6 +84,14 @@ statsPeriod: 60
 # add key: value map entries to customSettings to be added to pbgouncer.ini [pgbouncer] section
 customSettings: {}
 
+# Service configuration
+service:
+  enabled: true
+  type: ClusterIP
+  name: ""
+  port: 5432
```

Comment:
as mentioned above, moving the `port` here will cause problems for people who already have their own values overrides in existing installations / scripts. (even though this change is better)